src/Navbar.js

```diff
@@ -15,25 +15,33 @@ export default function Navbar({
   const previousClickHandler = dir === 'rtl' ? onNextClick : onPreviousClick;
   const nextClickHandler = dir === 'rtl' ? onPreviousClick : onNextClick;
 
-  const previousButton =
-    showPreviousButton &&
+  const previousButton = (
     <span
       role="button"
       aria-label={labels.previousMonth}
       key="previous"
-      className={classNames.navButtonPrev}
-      onClick={() => previousClickHandler()}
-    />;
+      className={
+        showPreviousButton
+          ? classNames.navButtonPrev
+          : `${classNames.navButtonPrev} DayPicker-NavButton-disabled`
+      }
+      onClick={showPreviousButton ? () => previousClickHandler() : null}
+    />
+  );
 
-  const nextButton =
-    showNextButton &&
+  const nextButton = (
     <span
       role="button"
       aria-label={labels.nextMonth}
       key="right"
-      className={classNames.navButtonNext}
-      onClick={() => nextClickHandler()}
-    />;
+      className={
+        showNextButton
+          ? classNames.navButtonNext
+          : `${classNames.navButtonNext} DayPicker-NavButton-disabled`
```

Comment:
Here and in the change above, please use `${classNames.navButtonInteractionDisabled}` and set it to `DayPicker-NavButton--interactionDisabled` in `classNames.js`.